### PR TITLE
refactor: ziofy SearchResponderV2 and Gravsearch (DEV-1755)

### PR DESCRIPTION
**Pull request by seakayone**
### Task Description/Number

Issue Number: DEV-

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [x] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/messages/util/search/QueryTraverser.scala [hunk 1]
```diff
@@ -5,24 +5,21 @@
...
-import akka.http.scaladsl.util.FastFuture
-import akka.pattern.ask
-
-import scala.concurrent.ExecutionContext
-import scala.concurrent._
-import scala.concurrent.duration._
+import zio.Task
+import zio.URLayer
+import zio.ZIO
+import zio.ZLayer
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/messages/util/search/QueryTraverser.scala:11]:
```suggestion
import zio._
```

File: webapi/src/main/scala/org/knora/webapi/messages/util/search/SparqlTransformer.scala [hunk 2]
```diff
@@ -5,9 +5,10 @@
 
 package org.knora.webapi.messages.util.search
 
+import zio.Task
+import zio.UIO
 import zio.ZIO
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/messages/util/search/SparqlTransformer.scala:10]:
```suggestion
import zio._
```
includes the next line, but GitHub doesn't let me annotate that

File: webapi/src/main/scala/org/knora/webapi/messages/util/search/gravsearch/types/GravsearchTypeInspectionUtil.scala [hunk 3]
```diff
@@ -5,21 +5,26 @@
 
 package org.knora.webapi.messages.util.search.gravsearch.types
 
-import scala.concurrent.ExecutionContext
+import zio.Task
+import zio.ZIO
+import zio.ZLayer
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/messages/util/search/gravsearch/types/GravsearchTypeInspectionUtil.scala:10]:
```suggestion
import zio._
```

File: webapi/src/main/scala/org/knora/webapi/responders/v2/ResponderWithStandoffV2.scala
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/responders/v2/ResponderWithStandoffV2.scala:1]: how come this could be deleted?

**Review by BalduinLandolt — APPROVED** [review]
LGTM

**Review by mpro7 — APPROVED** [review]

File: webapi/src/main/scala/org/knora/webapi/responders/v2/ResponderWithStandoffV2.scala
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/responders/v2/ResponderWithStandoffV2.scala:1]: The method was ziofied before and moved to `ConstructResponseUtilV2`.

**Pull request merged**